Rust/p005-ReLU-Activation.rs

```diff
@@ -0,0 +1,115 @@
+#[macro_use]
+
+extern crate ndarray;
+extern crate nalgebra;
+use ndarray::prelude::*;
+use ndarray::Array;
+use ndarray_rand::*;
+use ndarray_rand::rand_distr::*;
+use nalgebra::*;
+use std::num;
+use std::f64;
+use std::convert::From;
+
+struct Layer_Dense {
+	weights: Array2<f64>,
+	biases: Array2<f64>,
+	output: Option<Array2<f64>>,
+}
+impl Layer_Dense {
+ 	fn init(n_inputs: usize, n_neurons: usize) -> Self {
```

Comment:
Rust functions returning self, initializing struct should be named new. It's a python name convention to call those functions init